metrics-operator/converter/sli_converter.md

```diff
@@ -0,0 +1,66 @@
+# SLI Converter
+
+## Description
+
+SLI converter is a tool to convert `sli.yaml` files used in [KeptnV1](https://v1.keptn.sh/) into the new
+`AnalysisValueTemplate` resources used in the new kubernetes-native [Keptn](https://lifecycle.keptn.sh/).
+The converter is part of `metrics-operator` image.
+
+## Usage
+
+The converter will convert a single `sli.yaml` file into multiple `AnalysisValueTemplate` resources.
+
+To run the converter, execute the following command:
+
+```shell
+docker-run <METRICS_OPERATOR_IMAGE> manager --convert-sli=<PATH_TO_SLI> --sli-provider=<PROVIDER_NAME> --sli-namespace=<PROVIDER_NAMESPACE>
```

Comment:
This is just personal preference, but listing the required variables to be set at the beginning and then use them as env vars in the command makes it a bit easier to see immediately what needs to be set - I usually tend overlook and forget at least one value to be set if the placeholders are spread over the command ;) 

```suggestion
```shell
METRICS_OPERATOR_IMAGE=<METRICS_OPERATOR_IMAGE>
PATH_TO_SLI=<PATH_TO_SLI>
PROVIDER_NAME=<PROVIDER_NAME>
PROVIDER_NAMESPACE=<PROVIDER_NAMESPACE>

docker-run $METRICS_OPERATOR_IMAGE manager --convert-sli=$PATH_TO_SLI --sli-provider=$PROVIDER_NAME --sli-namespace=$PROVIDER_NAMESPACE
```